include/cantera/kinetics/KineticsFactory.h

```diff
@@ -48,7 +51,7 @@ Kinetics* newKineticsMgr(const string& model);
 shared_ptr<Kinetics> newKinetics(const string& model);
 
 /*!
- * Create a new kinetics manager, initialize it, and add reactions
+ * @brief Create a new kinetics manager, initialize it, and add reactions.
  *
  * @param phases     Vector of phases containing species which participate in
```

Comment:
For the sake of avoiding too much variety in the different ways of using Doxygen, you might consider the "autobrief" style that we use more frequently:
```cpp
//! Create a new kinetics manager, initialize it, and add reactions.
/*!
 * @param phases     Vector of phases containing species which participate in
```